src/guessNumber.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+/* eslint-disable no-console */
+
+const readline = require('readline');
+const { checkDigits } = require('./checkDigits');
+
+const inputInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const isValidGuess = (guess, neededNumber) => {
```

Comment:
The `isValidGuess` function should only check if the guess is valid, but should not log any messages. Remove the console.log statements and move them to another function if necessary.

Reply:
Or rename your function